### PR TITLE
Minor changes on the index page

### PR DESCRIPTION
**Pull request by bast**

### CODE REVIEW

File: content/index.rst [hunk 1]
```diff
@@ -23,19 +23,19 @@ branching.
 
 .. prereq::
 
-   -  A reasonably recent version of Git is installed. For installation
-      instructions see `the CodeRefinery installation
+   -  A reasonably recent version of **Git is installed**
```
Comment (rkdarst) [inline content/index.rst:26]: Do you know what our oldest requirement is?

* `git switch` is 2.23 (2019-08-16)
* `init.defaultBranch` is 2.28 (2020-07-27)

I don't think we do the second - do we do the first?

**Review by rkdarst — APPROVED** [review]
two suggestions

File: content/index.rst (hunk 1 above)
Comment (bast) [reply content/index.rst:26]: We haven't concluded on changing either yet so we postpone to another year.

Comment (bast) [reply content/index.rst:26]: So in other words we don't require reasonable recent version but it should work with almost any ancient version.

**Pull request merged**